App.jsx

```diff
@@ -22,8 +27,8 @@ const App = () => {
           setTasks(JSON.parse(storedTasks));
         } else {
           setTasks([
-            {id: 1, text: 'Doctor Appointment', completed: true},
-            {id: 2, text: 'Meeting at School', completed: false},
+            {id: 1, text: 'Doctor Appointment', status: 'todo'},
```

Comment:
use enum for status, instead of hardcoded strings. This handles the case if a fourth option gets added.